docs/HowTo/Store-Keys-Vaults/Use-AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+description: How to store keys in AWS Secrets Manager
+---
+
+# Use Web3Signer with AWS Secrets Manager
+
+Web3Signer supports storing BLS signing keys in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).
+
+The [AWS Secrets Manager documentation](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) provides
+the information you need to get started.
+
+## Store a private key in AWS Secrets Manager
+
+You need an AWS profile to use AWS Secrets Manager.
+
+Use the [Create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html)
+guide to store a new key in AWS Secrets Manager.
+
+Specify the following when
+[configuring the signing key configuration file](../Use-Signing-Keys.md#using-key-configuration-files):
+
+* Authentication mode.
+  Valid options are `ENVIRONMENT` and `SPECIFIED`.
+  If using `ENVIRONMENT`, specify the authentication credentials using the following environment variables:
```

Comment:
The default approach for the `ENIRONMENT` authentication mode is to follow the [setting credentials](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#setting-credentials) guide. A quick example from that guide would be to set the following environment variables.